e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js

```diff
@@ -56,36 +56,4 @@ describe('Messaging', () => {
             });
         });
     });
-
-    it('MM-T1797 Compact view: Show single image thumbnail', () => {
```

Comment:
This is duplicate of https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/messaging/message_spec.js#L168